modules/rules_xcodeproj/1.0.0rc1/presubmit.yml

```diff
@@ -0,0 +1,4 @@
+tasks:
+  verify_targets_macos:
+    name: Verify build targets
+    platform: macos
```

Comment:
Do I just remove this now that we don't have build or test targets?

Reply:
@Wyverald @fmeum

Reply:
probably fine to leave as-is.